src/js/components/alert.js

```diff
@@ -5,43 +5,67 @@
 
 export default class Alert {
 
-  constructor(options) {
-    this.handleClick = this.handleClick.bind(this);
+  constructor(element) {
 
-    this.init(options);
-  }
-
-  init(options) {
-    document.getElementById(options.id).closest('[aria-labelledby]').querySelector('[data-alert-close]').addEventListener('click', () => { this.handleClick(options.id); }, false);
-  }
+    // Instance properties
+    this.element = element;
+    this.closeAttribute = ['data-alert-close'];
+    this.closeSelector = `[${this.closeAttribute}]`;
+    this.ariaAttribute = ['aria-labelledby'];
```

Comment:
You can get rid of these two properties. Instead of using `aria-labelledby` as a hook to grab the actual alert element, let's look at using a data-attribute to bring it inline with the rest of the components.

I'll add some more comments about this, stand by...